config/default/kustomization.yaml

```diff
@@ -10,30 +6,7 @@ namespace: multicluster-gateway-controller-system
 # field above.
 namePrefix: mgc-
 
-# Labels to add to all resources and selectors.
-#commonLabels:
-#  someName: someValue
-
-# [CERTMANAGER] To enable cert-manager, uncomment all sections with 'CERTMANAGER'.
-#- ../certmanager
-# [PROMETHEUS] To enable prometheus monitor, uncomment all sections with 'PROMETHEUS'.
-#- ../prometheus
-
-# Protect the /metrics endpoint by putting it behind auth.
-# If you want your controller-manager to expose the /metrics
-# endpoint w/o any authn/z, please comment the following line.
-
-# [CERTMANAGER] To enable cert-manager, uncomment all sections with 'CERTMANAGER'.
-# Uncomment 'CERTMANAGER' sections in crd/kustomization.yaml to enable the CA injection in the admission webhooks.
-# 'CERTMANAGER' needs to be enabled to use ca injection
-#- webhookcainjection_patch.yaml
-
 resources:
-- ../crd
-- ../rbac
-- namespace.yaml
 - ../manager
 - ../add-on-manager
 - ../policy-controller
-patches:
-- path: manager_metrics_patch.yaml
```

Comment:
I would avoid changing this, and there really shouldn't be a need to. The point of default is it's an overlay that is specifying the namespace, prefix etc.. to use and calls on the individual parts (which should not specify a namespace)

This his how all kubebuilder controllers are set up see https://github.com/Kuadrant/kuadrant-operator/blob/main/config/default/kustomization.yaml